inc/mcp3564.h

```diff
@@ -247,6 +247,11 @@
 #define MCP3561_GAINCAL_SREAD   (MCP3561_GAINCAL_ADDR << _MCP3561_COMMAND_ADDR_POS) | MCP3561_SREAD_DATA_COMMAND
 #define MCP3561_GAINCAL_IREAD   (MCP3561_GAINCAL_ADDR << _MCP3561_COMMAND_ADDR_POS) | MCP3561_IREAD_COMMAND
 
+#define MCP3561_RSVD_ADDR       (0x0B)

```

Comment:
Maybe there is a more descriptive name instead of `RSVD`. I guess since nothing else uses anything "reserved" this is fine